doc/changes.rst

```diff
@@ -1,6 +1,111 @@
+* Add an example for plotting multi-parameter symbols (#772)
+* Add a gallery example for the fig.image() method (#777)
+* Add a gallery example of setting line colors with a custom CPT (#774)
+* Add a table for compatibility of PyGMT with Python and GMT (#763)
+* Add description for the "columns" arguments (#766)
+* Add table for projections (#753)
+* Add cylindric (#761), conic projection (#721, #757), miscellaneous projections descriptions (#723, #762), logarithmic and power projections (#742) to gallery
+* Add projection description for Lambert Azimuthal Equal-Area (#760)
+* Change text when GMTInvalidInput error is raised for basemap (#729)
+* Add non-geographic plots to projection gallery (#728)
+* Adding cylindrical projections to gallery (#727)
+* Add tutorials, for adding a map title (#720), plotting Earth relief (#712), 3D perspective image (#743), contour maps (#705), plotting lines tutorial (#741), region arguments (#800)
+* Add frame and cmap to the colorbar parameters list (#709)
+* Add frame as a parameter on the grdview documentation (#707)
+
+Bug Fixes
+
+* Fix a bug of Figure.text() when "text" is a non-string array (#724)
+* Fix the error message when IPython is not available (#701)
+
+Maintenance
+
+* Add dependabot to keep GitHub Actions up to date (#861)
+* Add authorship policy (#726)
+* Move plotting functions to separate modules (#808)
+* Skip workflows in PRs if only non-source-code files are changed (#839)
+* Move non-plotting modules to separate files in src directory (#832)
+* Show test execution times in pytest (#835)
+* Add slash command '/test-gmt-dev' to test GMT dev version (#831)
+* Add tests for grdfilter (#809)
+* Improve the error message when PyGMT fails to load the GMT library (#814)
+* Add tests for GMTInvalidInput for figure.py (#810)
+* Split meca into a standalone module from base_plotting.py (#686)
+* Update CONTRIBUTING.md with formatting conventions (#775)
+* Use args_in_kwargs for basemap error raising (#797)
+* Add docformatter to format plain text in docstrings (#642)
+* Add GMTInvalidInput Error for Figure.coast (#787)
+* Add args_in_kwargs function (#791)
+* Add a CI job testing GMT master branch on Windows (#756)
+* Add isort to sort imports alphabetically (#745)
+* Convert relative imports to absolute imports (#754)
+* Add a Makefile target 'distclean' for deleting project metadata files (#744)
+* Add test test_basemap_map_scale (#739)
+* Check files for UNIX-style line breaks and 644 permission (#736)
+* Rename vercel configuration file from now.json to vercel.json (#738)
+* Migrate pytest configurations to pyproject.toml (#725)
+* Migrate documentation deployment from Travis CI to GitHub Actions (#713)
+* Switch from versioneer to setuptools-scm (#695)
+* Migrate coverage configurations to pyproject.toml (#667)
+
+Deprecations
+
+* Drop Python 3.6 support (#699)
+
+Contributors
+
+@liamtoney, @meghanrjones, @michaelgrund, @seisman, @weiji14 and @willschlitzer
```

Comment:
```suggestion
People with a "+" by their name contributed a patch for the first time.

* `Dongdong Tian <https://github.com/seisman>`__
* `Liam Toney <https://github.com/liamtoney>`__
* `Meghan Jones <https://github.com/meghanrjones>`__ +
* `Michael Grund <https://github.com/michaelgrund>`__ +
* `Wei Ji Leong <https://github.com/weiji14>`__
* `Will Schlitzer <https://github.com/willschlitzer>`__ +
```

Two thoughts here:
1. Should we sort alphebetically by name (as above), or by number of commits (old style before).
2. Some projects (e.g. [geopandas](https://github.com/geopandas/geopandas/releases/tag/v0.8.0)) add a little note with a plus (+) to indicate people's first contribution, should we add it here?

Reply:
> Two thoughts here:
> 
> 1. Should we sort alphebetically by name (as above), or by number of commits (old style before).

I'm fine with both, but prefer "number of commits".

> 2. Some projects (e.g. [geopandas](https://github.com/geopandas/geopandas/releases/tag/v0.8.0)) add a little note with a plus (+) to indicate people's first contribution, should we add it here?

I saw it in many projects, but is there an easy way to know if someone makes his/her first contribution to this release?

Reply:
> > 1. Some projects (e.g. [geopandas](https://github.com/geopandas/geopandas/releases/tag/v0.8.0)) add a little note with a plus (+) to indicate people's first contribution, should we add it here?
> 
> I saw it in many projects, but is there an easy way to know if someone makes his/her first contribution to this release?

Other than remembering people, there's probably a complicated git command that could do it. Happy to leave it without the plus.

> I'm fine with both, but prefer "number of commits".

I thought your name would be on top either way, but `git shortlog HEAD...v0.2.1 -sne` gives:

```
    58  Will Schlitzer <schlitzer90@gmail.com>
    54  Dongdong Tian <seisman.info@gmail.com>
     9  Wei Ji <23487320+weiji14@users.noreply.github.com>
     7  Michael Grund <michael_grund@gmx.de>
     6  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
     3  Liam Toney <38269494+liamtoney@users.noreply.github.com>
     3  Meghan Jones <meghanj@hawaii.edu>
     3  Wei Ji <weiji.leong@vuw.ac.nz>
```
and so the order would be:
```suggestion
* `Will Schlitzer <https://github.com/willschlitzer>`__
* `Dongdong Tian <https://github.com/seisman>`__
* `Wei Ji Leong <https://github.com/weiji14>`__
* `Michael Grund <https://github.com/michaelgrund>`__
* `Liam Toney <https://github.com/liamtoney>`__
* `Meghan Jones <https://github.com/meghanrjones>`__
```

Reply:
Looks better to me.

Reply:
Alright, I'll commit the change, but we should document this (order by number of commits) for when we deal with #885.

Reply:
Running `git shortlog HEAD...v0.2.1 -sne` on the master branch gives me:
```
    54  Dongdong Tian <seisman.info@gmail.com>
    54  Will Schlitzer <schlitzer90@gmail.com>
     9  Wei Ji <23487320+weiji14@users.noreply.github.com>
     7  Michael Grund <michael_grund@gmx.de>
     6  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
     3  Liam Toney <38269494+liamtoney@users.noreply.github.com>
     3  Meghan Jones <meghanj@hawaii.edu>
```

Not trying to be listed at the top, just curious why our results are different 😂

Reply:
Hmm, I was probably on a different branch (which explains why it had my unversity vuw.ac.nz email). I think you'll get bumped down after @willschlitzer merges this PR though :rofl:

Reply:
Dare I suggest that as a GMT developer and long-time PyGMT maintainer, I think @seisman has definitely earned the first spot!